cocos/gi/light-probe/delaunay.ts

```diff
@@ -194,30 +213,27 @@ export class Delaunay {
     private _probes: Vertex[] = [];
     private _tetrahedrons: Tetrahedron[] = [];
 
+    private _triangles: Triangle[] = [];
+    private _edges: Edge[] = [];
+
     public getProbes () { return this._probes; }
     public getTetrahedrons () { return this._tetrahedrons; }
 
-    public build (points: Vec3[]) {
-        this.reset();
-
-        const pointCount = points.length;
-        if (pointCount < 4) {
-            warnID(17000);
-            return;
-        }
-
-        for (let i = 0; i < points.length; i++) {
-            this._probes.push(new Vertex(points[i]));
-        }
+    public build (probes: Vertex[]) {
+        this._probes = probes;
```

Comment:
In c++, `probes` is copied. Here `this._probes` is the same as input `probes`. 
The `probes`'s content might be changed outside.

Reply:
Yes, two implementations are all right. Delaunay object is used temporarily. But c++'s implementation should be improved later.